### ISSUE TITLE
WPS317 is inconsistent with PEP8 in function definition

### ISSUE THREAD

**Issue by cyyc1** [wemake-services/wemake-python-styleguide#2498]
### What's wrong

Hi, maintainers!

The recommended style in WPS317, [ParametersIndentationViolation](https://wemake-python-styleguide.readthedocs.io/en/latest/pages/usage/violations/consistency.html#wemake_python_styleguide.violations.consistency.ParametersIndentationViolation), for function definition is:

```python
def my_function(
    arg1,
    arg2,
    arg3,
) -> None:
    return None
```

However, PEP8 (https://peps.python.org/pep-0008/#indentation) recommends adding 4 spaces to distinguish arguments from the rest (see the screenshot below):

![image](https://user-images.githubusercontent.com/114281716/192087123-0ea64ebc-f5d3-4a0e-8065-b5fc04d87c44.png)

And it also explicitly say that not adding 4 extra spaces is wrong:

![image](https://user-images.githubusercontent.com/114281716/192087092-50c0b7f7-3cea-4073-9389-91b80c97c535.png)

### How it should be

WPS317 should allow 4 extra spaces after function definition

### Flake8 version and plugins

N/A (I was using your playground)

### pip information

N/A (I was using your playground)

### OS information

N/A (I was using your playground)

**Comment by sobolevn** [comment]
We are fine with being inconsistent with any other styleguides: we only care about consistency within WPS.
Thanks for raising this!

**Comment by Corrosion667** [comment]
@sobolevn Do you mean that PEP-8 is not an authoritative source of code style rules?

**Comment by sobolevn** [comment]
> Do you mean that PEP-8 is not an authoritative source of code style rules?

Yes, this is why we had to come up with our own set of rules.
